plugins/woocommerce/client/blocks/package.json

```diff
@@ -69,7 +69,7 @@
 		"lint:js-fix": "eslint assets/js --ext=js,jsx,ts,tsx --fix",
 		"lint:md:docs": "wp-scripts lint-md-docs",
 		"pre-commit": "lint-staged",
-		"reformat-files": "prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json,ts,tsx}\"",
+		"reformat-files": "prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,ts,tsx,css,scss}\"",
```

Comment:
Out of curiosity, why was `json` removed from the list?

Reply:
Because it is ignored in `.prettierignore` [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/blocks/.prettierignore#L1). Felt sensible to align the two sources.